### DISCUSSION TITLE
Question: How to download week 7

### DISCUSSION THREAD

**Discussion by JohnyDawn** [FunkinCrew/Funkin#1532] [General]
Hi, I was just asking how exactly do you download week 7. It's been really confusing and I'm not a big programmer. So basically I just need help with downloading week 7. 

Sincerely,
JohnyDawn

**Comment by xiiixvv** [comment]
Not know what "Exclusive" means do ya

**Comment by JohnyDawn** [comment]
> Ok, when the version says Newgrounds Exclusive, and when it's newgrounds, it's about the big and good ol flash games and flash videos, right?
> There's no way to download week 7 nuff said.

???????????

**Comment by CaptainGame17** [comment]
you cannot

**Comment by Awfyboy** [comment]
You can't, it's a Newgrounds exclusive. So it's only available online. The itch.io version will be released soon, so wait until that one releases.

**Reply by isakube** [reply]
ninja confirmed as answer??????? ITCH IO RELEASE SOON???????????

**Comment by foxsouns** [comment]
> You can't, it's a Newgrounds exclusive. So it's only available online. The itch.io version will be released soon, so wait until that one releases.

and afaik, the source will fall with it. ninja said on twitter that he'll give the source/itch binaries out if either of these happen
1. funkin on newgrounds becomes the first search on google; or
2. funkin becomes number 1 on newgrounds 

time to wait

**Reply by DOGZ1234** [reply]
https://www.google.com/search?q=funkin+on+newgrounds&rlz=1C5CHFA_enUS951US951&oq=funkin+on+newgrounds&aqs=chrome..69i57j0i22i30j69i60.225j0j7&sourceid=chrome&ie=UTF-8

**Reply by ACrazyTown** [reply]
> https://www.google.com/search?q=funkin+on+newgrounds&rlz=1C5CHFA_enUS951US951&oq=funkin+on+newgrounds&aqs=chrome..69i57j0i22i30j69i60.225j0j7&sourceid=chrome&ie=UTF-8

yeah it'll be the first result when you google "funkin on newgrounds", pretty sure they meant that googling "funkin" or "friday night funkin" will result in the NG link being the first

**Comment by JohnyDawn** [comment]
ok i guess wait a week

**Comment by aflacc** [comment]
if you google friday night funkin and the newgrounds page is the top result you will be able to downolad it

**Comment by catte-cloud** [comment]
> Hi, I was just asking how exactly do you download week 7. It's been really confusing and I'm not a big programmer. So basically I just need help with downloading week 7.
> 
> Sincerely,
> JohnyDawn

You can't.

**Comment by foxsouns** [comment]
Quite literally: you must wait until Ninja releases the source code/new binaries; else play it on newgrounds.

**Comment by GabrielSomething** [comment]
fuck the old message im fucking dumb,
game is LIMITED exclusive to web only, so you just gotta wait.
sorry for like confusing everyone.

WHY DID I SAY EXCLUSIVE TO PC JFKDFJKFJKFDJKDJKDFJK
